app/controllers/account_controller.rb

```diff
@@ -330,6 +333,8 @@ def prefs
     update_copyright_holder if prefs_changed_successfully
   end
 
+  # alias :show :prefs # maybe? or should profile be a :show? - AN
```

Comment:
I've always been confused by the organization of `account` vs `user`. I wish I knew how other applications typically handle this stuff.
To me, `profile` feels like it belongs to `user` ... something like `user/show` (with some attributes displayed only if in admin_mode or if viewing one's own profile).
But that's probably a set of changes that should wait until all the tests are passing with the current functionality.

Reply:
Yes - i'm not proposing changing that division of function across those two user controllers, although we can revisit later of course. The way i'm understanding the current division is: 
Account: "manage your own stuff"
User: "manage other people's stuff" (for admins)

The reason i aliased Account#index to :login was to get the navbar "context home" title link working quickly, but i believe this will need to be fixed properly to get the navbar's intended UI working. What i'm proposing there is a bit like the use of "model.show_controller" - 

The navbar as i've currently got it designed needs the controller to provide a link (with a translatable title) so the user can re-orient themselves in the current context of their activity. In Account, it should probably be login, or their profile. In Observations, it would probably be the index. Here's a screenshot of how it appears in the navbar.

![Screen Shot 2020-07-12 at 8 43 47 AM](https://user-images.githubusercontent.com/1948095/87250735-ded6c680-c41b-11ea-9c29-2756771c8780.png)